examples/demo/site/modules/auth/mail_templates/account_evaluation.html

```diff
@@ -0,0 +1,26 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Account Evaluation</title>
+  <meta name="description" content="Account Evaluation">
+  <meta name="author" content="ROC CMS">
+</head>
+
+<body>
+    <h2> Account Evaluation </h2>
+	<p>The user $user ($email) wants to register to the site <a href="$host">ROC CMS</a></p>
+
+	<blockquote><p>This is his/her application.</p>
+	  <p>$application</p>
+	</blockquote>
+
+	<p>To complete the registration, please click on the following link to activate the user account:<p>
+
+	<p><a href="$activate">$activate</a></p>
```

Comment:
$activation_url  would be a better var name
same  for $rejection_url
and probably  $host_url  or $siteurl   to follow ROC CMS terms.